### ISSUE TITLE
train other contrast methods

### ISSUE THREAD

**Issue by Monmeng** [SirisAvishek/Attention_Shift_Ranks#8]
hello,Can you tell me how the results of other comparison methods were obtained? For example BASNet,CPD-R.These methods are all  binary segmentation SOD. So how do you get the subjective contrast in the paper? 
Thanks

**Comment by Monmeng** [comment]
Can you provide the test results of other comparison methods on ASSR dataset, or other trained model codes of compares methods.   Thank you very much

**Comment by SirisAvishek** [comment]
@Monmeng, hi, the process for obtaining the results from comparison methods should be mentioned in the paper. We use the GT segmentations to extract pixels for each GT object from the predicted saliency maps. Then, we average a score for each object based on the value of their pixels. The ranks of each object is then obtained by ordering their scores. 

Here is a google drive link to the results of the comparison methods on our dataset.
https://drive.google.com/file/d/1p7be6-QQfKYlLToUuEiCB7tU3MKi27qy/view?usp=sharing

**Comment by Monmeng** [comment]
Thank you for your reply. It helps me a lot. But I have a question, how to get the predicted saliency maps of BASNet and CPD-R ? Did you retrain their models with the ASSR train set, or did you test the ASSR test set directly with the training parameters they provided? If you're retraining,  what's the loss function that you're using? My personal feeling is that we can't use the cross entropy function, because GT is not a binary map.
The above is what confused me. Thank you again for your recovery in your busy schedule.
